scripts/rssFeedGen.py

```diff
@@ -0,0 +1,95 @@
+from bs4 import BeautifulSoup
+from datetime import datetime
+import time
+from email.Utils import formatdate
+import PyRSS2Gen
+import StringIO
+import re
+
+def extract_releases(html):
+    html_doc = open(html, 'r').read()
+    soup = BeautifulSoup(html_doc, "lxml")
+    divRelease = soup.find("div", {"id":"release-notes"})
+    releases = divRelease.findAll("div", {"id":re.compile("release-")})
+    return releases
+
+def id_to_pubDate(r):
+    id_str = r.findNext("span").get("id")
+    date_str = ''
+    for i in id_str.split():
+        for l in i:
+            if l.isdigit():
+                date_str = date_str + l
+    date_str = date_str + " 00:00:00"
+    date_obj = datetime.strptime(date_str, '%Y%m%d %H:%M:%S')
+    date_rss = datetime.strftime(date_obj, "%a, %d %b %Y %H:%M:%S %z")
+    return date_rss
+
+def extract_api_changes(r):
+    api_changes = r.findNext("a",{"href":re.compile("https://github.com/geoadmin/mf-chsdi3")}, text="Full changelog")
```

Comment:
I would remove the text arg and use `https://github.com/geoadmin/mf-chsdi3/compare/` until compare. It would be slightly more robust.